core/templates/dev/head/components/CKEditorDirective.js

```diff
@@ -0,0 +1,219 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Rename this file to `CkEditorRteDirective.js`

Reply:
done!